### ISSUE TITLE
[BUG] History wrong price calculation in Liquid

### ISSUE THREAD

**Issue by znerol19619** [CoinAlpha/hummingbot#1415]
**Describe the bug**
History price doesn't match the active order price before it gets filled. I've only encounter this behavior in liquid so far.

**Steps To Reproduce**

1. Setup PMM or XEMM
2. Choose Liquid as exchange.
3. Run ``status`` command, and check the active order price. 
4. Wait for an order to get filled. 
5. Run ``history`` or ``list trades`` command.
...

**Screenshots**
![Screenshot from 2020-02-13 16-03-58](https://user-images.githubusercontent.com/27669951/74419629-30871f00-4e85-11ea-9757-65b0be113acc.png)
![Screenshot from 2020-02-13 16-03-38](https://user-images.githubusercontent.com/27669951/74419652-354bd300-4e85-11ea-97f2-8d3a6e05ca56.png)

**Release version**
```
                "branch": "development",
                "commit": "8b3a8e9bfc4239c1549b626deef9c12c9e28ec51",
                "date": "2020-02-05T00:39:44Z",

```

**Attachments**
[liq_qash-eth.txt](https://github.com/CoinAlpha/hummingbot/files/4197438/liq_qash-eth.txt)
[logs_conf_pure_market_making_strategy_0.log](https://github.com/CoinAlpha/hummingbot/files/4197452/logs_conf_pure_market_making_strategy_0.log)

**Comment by Hashira-Lars** [comment]
One of the user reported this experiencing with XEMM liquid maker.

**Comment by dennisocana** [comment]
I also ran into this error while triaging another issue in Liquid. Based on the logs, the prices are correct when trades are executed. However, the price is incorrect when displayed in `history` output.

<img width="463" alt="1qxjrbpWVs" src="https://user-images.githubusercontent.com/50150287/78527438-6dc9b680-780f-11ea-81e7-3bb7aefacc1c.png">

<img width="773" alt="ddDStI07Q7" src="https://user-images.githubusercontent.com/50150287/78527452-74582e00-780f-11ea-9661-dc4b064997e2.png">

Trade 1 :

> 2020-04-06 05:54:02,000 - 10 - hummingbot.strategy.pure_market_making.pure_market_making_v2 - INFO - (QASH-ETH) Creating limit ask orders at (Size, Price): ['3E+1 QASH, 0.00024588 ETH'] [clock=2020-04-06 05:54:02+00:00]

Trade 2 :

> 2020-04-06 05:54:40,909 - 10 - hummingbot.strategy.pure_market_making.pure_market_making_v2 - INFO - (QASH-ETH) Maker sell order sell-QASH-ETH-1586152473001993 (30.0 QASH @ 0.00024432 ETH) has been completely filled. [clock=2020-04-06 05:54:40+00:00]

[logs_pmm_liquid_qash-eth.log](https://github.com/CoinAlpha/hummingbot/files/4436064/logs_pmm_liquid_qash-eth.log)

`dev-0.26.0`

**Comment by dennisocana** [comment]
Fixed by https://github.com/CoinAlpha/hummingbot/pull/1590 in development branch and will be available in the upcoming `0.26.0` release.
